src/app.html

```diff
@@ -6,6 +6,10 @@
     <meta name="viewport" content="width=device-width" />
     <link rel="stylesheet" href="%sveltekit.assets%/app.css" />
     <link rel="stylesheet" href="%sveltekit.assets%/menu.css" />
+    <link rel="stylesheet" href="%sveltekit.assets%/fontawesome/css/fontawesome.css" />
+    <link rel="stylesheet" href="%sveltekit.assets%/fontawesome/css/brands.css" />
+    <link rel="stylesheet" href="%sveltekit.assets%/fontawesome/css/solid.css" />
+    <link rel="stylesheet" href="%sveltekit.assets%/fontawesome/css/regular.css" />
```

Comment:
Including the css in here was more performant that including the svg/js in here. The icons would not appear for a few milliseconds until the page loaded, which looked janky.

Evidently there's some wackiness people have been seeing when attempting to animate css icons, though, so something to look out for if I'm trying to use fontawesome's animation classes.